connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/arrow/ArrowEncoderSuite.scala

```diff
@@ -776,6 +776,16 @@ class ArrowEncoderSuite extends ConnectFunSuite with BeforeAndAfterAll {
     }
   }
 
+  test("kryo serialization") {
```

Comment:
Oh, this newly added test case failed. Could you take a look at the failure?

```
[info] - kryo serialization *** FAILED *** (3 milliseconds)
[info]   Expected exception org.apache.spark.SparkException to be thrown, but org.apache.spark.SparkRuntimeException was thrown (ArrowEncoderSuite.scala:780)
```

Reply:
yeah definitely my bad. Will update tonight.